src/settings/command_arguments.rs

```diff
@@ -63,13 +55,66 @@ pub struct CommandArguments {
     /// for the node's blockchain
     #[structopt(long = "genesis-config", parse(from_os_str))]
     pub genesis_data_config: PathBuf,
+}
+
+#[derive(StructOpt, Debug)]
+pub struct InitArguments {
+    /// set the address that will have all the initial funds associated to it.
+    /// This is the wallet that will serve as faucet on testnets and as initial
+    /// coin wallet for mainnets
+    #[structopt(long = "initial-address", parse(try_from_str))]
+    pub initial_address: AddressReadable,
+
+    /// set the time between the creation of 2 blocks. Value is a positive integer to
+    /// be in seconds.
+    #[structopt(
+        short = "slot-duration",
+        parse(try_from_str = "parse_duration"),
+        default_value = "15"
+    )]
+    pub slot_duration: std::time::Duration,
+
+    /// set the number of blocks that can be used to pack in the storage
+    #[structopt(long = "epoch-stability-depth", default_value = "2600")]
+    pub epoch_stability_depth: usize,
+}
+
+#[derive(StructOpt, Debug)]
+#[structopt(
+    name = "jormungandr",
+    raw(setting = "structopt::clap::AppSettings::ColoredHelp")
+)]
+pub struct CommandLine {
+    /// activate the verbosity, the more occurrences the more verbose.
+    /// (-v, -vv, -vvv)
+    #[structopt(short = "v", long = "verbose", parse(from_occurrences))]
+    pub verbose: u8,
 
     /// Set format of the log emitted. Can be "json" or "plain"
-    #[structopt(long = "log-format", parse(try_from_str))]
-    pub log_format: Option<LogFormat>,
+    #[structopt(long = "log-format", parse(try_from_str), default_value = "plain")]
+    pub log_format: LogFormat,
+
+    #[structopt(subcommand)]
+    pub command: Command,
+}
+
+#[derive(StructOpt, Debug)]
+pub enum Command {
+    /// start jormungandr service and start participating to the network
+    #[structopt(name = "start")]
+    Start(StartArguments),
+
+    /// initialize a configuration
+    #[structopt(name = "init")]
+    Init(InitArguments),
```

Comment:
Maybe use `InitConfig` for clarity?

Reply:
I'd rather like to remain consistent with the other option. Especially that we are still in the command line arguments module here.